static/app/components/modals/debugFileCustomRepository/appStoreConnect/stepOne.tsx

```diff
@@ -52,14 +52,19 @@ function StepOne({stepOneData, onSetStepOneData}: Props) {
       >
         <Textarea
           name="privateKey"
-          placeholder={t('Private Key')}
           value={stepOneData.privateKey}
           rows={5}
           autosize
+          placeholder={
+            stepOneData.unchanged
+              ? `${'('}${t('Private Key unchanged')}${')'}`
```

Comment:
```suggestion
              ? `(${t('Private Key unchanged')})`
```

Or why not

```suggestion
              ? t('(Private Key unchanged)')
```

Reply:
I was told that I should not use those in a translation 🤷‍♀️

Reply:
oh interesting. Do you know why? The first change is probably fine in that case

Reply:
I can't remember the now... I will update the code and bring this topic to the next tsc meeting 😉

Reply:
I think the answer I got was "that is something we can't translate, so it's better to have outside of the function"